doc/api/tls.md

```diff
@@ -805,6 +856,21 @@ and their processing can be delayed due to packet loss or reordering. However,
 smaller fragments add extra TLS framing bytes and CPU overhead, which may
 decrease overall server throughput.
 
+### tlsSocket.ssl.verifyError()
+<!-- YAML
+added: REPLACEME
+-->
+
+Returns an `Error` object if the peer's certificate fails validation.
+
+Validation includes many checks, including verification that the certificate
```

Comment:
To avoid `includes... including`, maybe something like:
```suggestion
Validation contains many checks, including verification that the certificate
```